blocks/recentreports/recentreports.css

```diff
@@ -0,0 +1,168 @@
+.block.recentreports .row {
+  display: flex;
+  flex-wrap: wrap;
+  margin-right: -0.9rem;
+  margin-left: -0.9rem; 
+}
+
+.block.recentreports .box h3 {
+  font-size: var(--body-font-size-m);
+  margin-bottom: var(--space-12px);  
+  height: var(--heading-font-size-xxl);
+} 
+
+.block.recentreports .positive {
+  color: var(--color-green); 
+}
+
+.block.recentreports .box h3 a {
+  color: var(--primary);
+  height: var(--heading-font-size-xxl);
+}
+
+.block.recentreports .box {
+  border: var(--line-width-hairline) solid var(--color-light-gray-dark);
+  border-radius: var(--space-8px);
+  padding: 1.25rem 1rem 4.06rem;
+  position: relative;
+  max-width: 16.5rem; 
+  height:100%;
+}
+
+.block.recentreports .discover-more {
```

Comment:
should this discover-more be uplevel & put it [in style.css here](https://github.com/aemsites/icicidirect/blob/main/styles/styles.css#L471)?

Reply:
fixed